scripts/obtain/dotnet-install.ps1

```diff
@@ -0,0 +1,338 @@
+#
```

Comment:
Why are these two files diffing as completely new?  Did you remove the old install.ps1/sh?